dashboard.js

```diff
@@ -0,0 +1,10 @@
+const cookieExist=document.cookie;
+const cookieDeatails=cookieExist.split("=")
+if (cookieDeatails[1]===undefined){
```

Comment:
don't do explicit type-checking. negate the value to make it truthy
```suggestion
if (!cookieDeatails[1]){
```